cli/search/search.go

```diff
@@ -115,6 +124,49 @@ func GetVersionsFromGitRemote(repo string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// GetCommitsFromGitRemote returns commits list from specified remote git repo.
+func GetCommitsFromGitRemote(repo string) ([]string, error) {
+	commits := []string{}
+
+	if _, err := exec.LookPath("git"); err != nil {
+		return nil, fmt.Errorf("'git' is required to search among existing commits")
+	}
+
+	tempRepoPath, err := os.MkdirTemp("", "versions_log")
+
+	if err != nil {
+		return nil, fmt.Errorf("failed to get commits from %q: %s", repo, err)
+	}
+
+	defer os.RemoveAll(tempRepoPath)
+
+	cmd := exec.Command("git", "clone", "--filter=blob:none", "--no-checkout",
+		"--single-branch", repo, tempRepoPath)
+
+	err = cmd.Run()
+
+	if err != nil {
+		return nil, err
+	}
+
+	cmd = exec.Command("git", "log", "--oneline")
```

Comment:
Looks like this is the same logic as in `GetCommitsFromGitLocal`. Why don't we just call `GetCommitsFromGitLocal` from here providing cloned repo path?